source/fundamentals/object-serialization.txt

```diff
@@ -0,0 +1,242 @@
+====================
+Object Serialization
```

Comment:
This covers both serialization and deserialization to and from Bson to Java types.